src/main/java/animals/controller/Game.java

```diff
@@ -0,0 +1,147 @@
+package animals.controller;
+
+import animals.localization.LanguageRule;
+import animals.localization.LanguagesRulesEn;
+import animals.localization.LanguagesRulesEo;
+import animals.model.Node;
+import animals.service.FileService;
+import animals.service.KnowledgeTreeService;
+import animals.service.MenuService;
+import animals.service.ResourceBundleService;
+
+import java.time.LocalTime;
+import java.util.Scanner;
+
+import static animals.service.ResourceBundleService.getLocalString;
+
+public class Game {
+    private static final Scanner SCANNER = new Scanner(System.in);
+    private static final LocalTime MORNING_START = LocalTime.of(5, 0);
+    private static final LocalTime DAY_END = LocalTime.of(17, 59);
+
+    public FileService fs;
+    public KnowledgeTreeService kts = new KnowledgeTreeService();
+    public LanguageRule lr = getLanguageRule();
```

Comment:
Why is FileService initialised in constructor but KnowledgeTreeService and LanguageRule is not?
Please check that all class fields are initialised in constructor. This class is not the one place.